types/chrome/index.d.ts

```diff
@@ -1593,7 +1593,14 @@ declare namespace chrome.declarativeContent {
     }
 
     /** Declarative event action that shows the extension's page action while the corresponding conditions are met. */
-    export class ShowPageAction { }
+	export class ShowPageAction { }
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/chrome/index.d.ts;
  Line: 1596;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.